src/bin/pg_autoctl/cli_create_drop_node.c

```diff
@@ -64,19 +64,21 @@ CommandLine create_postgres_command =
 	make_command("postgres",
 				 "Initialize a pg_auto_failover standalone postgres node",
 				 "",
-				 "  --pgctl                 path to pg_ctl\n"
-				 "  --pgdata                path to data director\n"
-				 "  --pghost                PostgreSQL's hostname\n"
-				 "  --pgport                PostgreSQL's port number\n"
-				 "  --listen                PostgreSQL's listen_addresses\n"
-				 "  --username              PostgreSQL's username\n"
-				 "  --dbname                PostgreSQL's database name\n"
-				 "  --nodename              pg_auto_failover node\n"
-				 "  --formation             pg_auto_failover formation\n"
-				 "  --monitor               pg_auto_failover Monitor Postgres URL\n"
-				 "  --auth                  authentication method for connections from monitor\n"
-				 "  --candidate-priority    priority of the node to be promoted to become primary\n"
-				 "  --replication-quorum    true if node participates in write quorum\n"
+				 "  --pgctl                 			path to pg_ctl\n"
+				 "  --pgdata                			path to data director\n"
+				 "  --pghost                			PostgreSQL's hostname\n"
+				 "  --pgport                			PostgreSQL's port number\n"
+				 "  --listen                			PostgreSQL's listen_addresses\n"
+				 "  --username              			PostgreSQL's username\n"
+				 "  --dbname                			PostgreSQL's database name\n"
+				 "  --nodename              			pg_auto_failover node\n"
+				 "  --formation             			pg_auto_failover formation\n"
+				 "  --monitor               			pg_auto_failover Monitor Postgres URL\n"
+				 "  --auth                  			authentication method for connections from monitor\n"
+				 "  --candidate-priority    			priority of the node to be promoted to become primary\n"
+				 "  --replication-quorum    			true if node participates in write quorum\n"
+				 "  --run         						create node then run pg_autoctl service\n"
+				 "  --exit-if-postgres-is-not-running   exit pg_autoctl if postgres node can not be started\n"
```

Comment:
Maybe we could keep the previous alignment and have the `--exit-...` option that is very long be off-alignment?

I'm not sure what the standard terminal width is nowadays, but I now I like it better when it's easy to copy/paste text from the terminal to an email or a document and have it fit on the page, which to this days still mean 80 columns, or in the email case if you want to make sure it fits, 72 columns...